.github/workflows/esm_tests_computer_workflow.yml

```diff
@@ -30,46 +30,32 @@ jobs:
       LC_ALL: en_US.UTF-8
       LANG: en_US.UTF-8
     container:
-      image: koldunovn/esm_test:latest
+      image: mandresm/ubuntu-miniforge-esm_tests:v1.0
```

Comment:
Change to ghcr hosting at some point in the future.

Reply:
```suggestion
      # TODO (PG): Change to ghcr hosting at some point in the future.
      image: mandresm/ubuntu-miniforge-esm_tests:v1.0
```